src/legiscrapor/nlpIE.py

```diff
@@ -19,9 +20,14 @@ def load_lang_model(lang):
     ## pick the correct spacy NLP library
     global nlp 
     if lang == "English":
-       nlp = spacy.load('en_core_web_sm',disable=['ner','textcat'])
+        try:
```

Comment:
much classier exception handling than what I had. Thanks for this.

Reply:
=)

Reply:
first time ever encountering this kind of thing myself.